models/core-java/src/main/java/org/commonjava/indy/model/core/StoreKeyPaths.java

```diff
@@ -0,0 +1,50 @@
+package org.commonjava.indy.model.core;
```

Comment:
Does this class have applicability outside of the promote add-on? If not, can we move this (and the CDI producer logic for it) into the promote add-on?

Reply:
No other place using it. It is safe to move it to promote add-on. I am not sure whether to use it outside of promotion in the future. But we can always expose it when it is needed then.